app/controllers/restaurants_controller.rb

```diff
@@ -4,6 +4,7 @@ def index
   end
 
   def show
+    @cart = Cart.find(session[:cart_id]) if session[:cart_id]
```

Comment:
I would suggest extractin this line to a separate method and call in in a 'before_action'. Check rails documentation to read more about it

Reply:
We'll try